src/tmpfiles/tmpfiles.c

```diff
@@ -3724,6 +3733,14 @@ static int parse_line(
                                 CREATE_SUBVOLUME_INHERIT_QUOTA,
                                 CREATE_SUBVOLUME_NEW_QUOTA) ? 0755 : 0644;
 
+        if (missing_user_or_group && (i.mode & ~0777)) {
```

Comment:
doesnt' matter, but as per coding style we should not rely on C's downgrade-to-bool logic for the mode comparison != 0 because it's not boolean or pointer, but an arithmetic value, i..e.

```c
… && (i.mode & ~0777) != 0)) {
``